atest/robot/standard_libraries/builtin/repeat_keyword.robot

```diff
@@ -25,11 +25,19 @@ Zero And Negative Times
     ${tc} =    Check Test Case    ${TEST NAME}
     Check Repeated Messages    ${tc.kws[0]}    0
     Check Repeated Messages    ${tc.kws[2]}    0
+    Check Repeated Messages With Time    ${tc.kws[3]}    0
```

Comment:
Shouldn't you be able to use the same keyword here as the earlier steps?

It would actually be a good idea to add a separate test for not executing keyword at all if time is 0 or smaller. The old test can be left for testing the same behavior when count <= 0.